src/helpers/index.tsx

```diff
@@ -21,4 +21,23 @@ export const getTextTimeUnit = (value: number, unit : ITimeUnit) => {
       default:
         return value < 2 ? `${value} Hour` : `${value} Hours`;
     }
-}
\ No newline at end of file
+};
+
+
+export const getImageType = (img: HTMLImageElement) => {
+  const dataType = img.src.substr(0, 20);
+  if (dataType.includes('data')) {
+    return dataType.split('/')[1].split(';')[0];
+  }
+  return 'image/jpeg';
+};
```

Comment:
Well this is based on the Mime Types

https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types

I'm using this helper to determinate the type of the image base and put the same in the return new image created.

Since I'm drawing the image in the canvas. If I draw an image that was a png  I wan to crop and get a png. If I receive a jpeg, I want to crop and obtain a jpeg.. if I receive any other kind of image (but it has to be an image) I will generate a jpeg.

https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/toDataURL

I kind of remembered that you mentioned that we were limiting to jpeg and png for the AvatarUploader, but wasn't sure it has to be the same for the Croptool or if we can receive other types
https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Image_types

In any case I'm validating that we receive from the drop file and the input file a valid image jpeg, png.. I could add more if is expected.

```js
const isValidImage = (file: File) => {
  const acceptedImageTypes = ['image/jpeg', 'image/png'];

  return file && acceptedImageTypes.includes(file['type']);
};
```